myhoard/util.py

```diff
@@ -683,3 +690,18 @@ def file_name_for_basebackup_split(base_file_name: str, split_nr: int) -> str:
         return f"{base_file_name}.{split_nr:03d}"
     else:
         return base_file_name
+
+
+def find_extra_xtrabackup_executables() -> list[BinInfo]:
+    raw_paths = os.environ.get("PXB_EXTRA_BIN_PATHS")
+    if not raw_paths:
+        return []
+    result = []
+    for extra_raw_path in raw_paths.split(os.pathsep):
+        extra_path = Path(extra_raw_path)
+        if extra_path.is_dir():
```

Comment:
Can't we assume it's always directory as env name suggests? I find it non intuitive if binary paths are provided and in that case we also need to add validation that the binary actually is xtrabackup.